djoser/views.py

```diff
@@ -130,11 +130,12 @@ def _action(self, serializer):
     def get_users(self, email):
         if self._users is None:
             email_field_name = get_user_email_field_name(User)
-            active_users_kwargs = {
-                email_field_name + '__iexact': email, 'is_active': True
+            filter_users_kwargs = {
+                email_field_name + '__iexact': email
             }
-            active_users = User._default_manager.filter(**active_users_kwargs)
-            self._users = [u for u in active_users if u.has_usable_password()]
+            _users = User._default_manager.filter(**filter_users_kwargs)
```

Comment:
There is no need to start local variable name with underscore. In my opinion it is only meaningful in class/instance scope and indicates if a member should be treated like a private.